core/storage/statistics/gae_models.py

```diff
@@ -613,7 +788,8 @@ def _insert_submitted_answers_unsafe(
             last_shard.submitted_answer_list,
             last_shard.accumulated_answer_json_size_bytes,
             new_submitted_answer_dict_list)
-        new_shard_count = main_shard.shard_count + len(sharded_answer_lists) - 1
+        new_shard_count = main_shard.shard_count + (
```

Comment:
Why did you break the line? Was it exceeding the 80 character limit?

Reply:
yeah, it was.